### PR TITLE
LPS-185740

### PR DESCRIPTION
**Pull request by michaelmdelcavalcanti**

### CODE REVIEW

**Comment by michaelmdelcavalcanti** [comment]
ci:test:sf

**Comment by michaelmdelcavalcanti** [comment]
ci:test:relevant

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCheck.java [hunk 1]
```diff
@@ -54,15 +55,28 @@ private String _fixImports(JavaClass javaClass, String content)
...
-					StringBundler.concat("import ", newImportName, ";"));
+				if (_isClassNameEquals(importName, newImportName)) {
+					content = _replaceImport(
+						content, importName, newImportName);
+				}
+				else {
+					content = _replaceImport(
+						content, importName, newImportName);
+
+					content = _replaceImpl(content, _className, _newClassName);
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCheck.java:66]: @michaelmdelcavalcanti, the second and third parameters of this method are actually attributes from this class, in light of this why don't we invoke them directly from the method implementation and keep just the first parameter?

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCheck.java [hunk 2]
```diff
@@ -97,6 +111,54 @@ private Map<String, String> _getMap(String fileName) throws Exception {
...
+	}
+
+	private String _replaceImport(
+		String content, String importName, String newImportName) {
+
+		return StringUtil.replace(
+			content,
+			StringBundler.concat("import ", importName, StringPool.SEMICOLON),
+			StringBundler.concat(
+				"import ", newImportName, StringPool.SEMICOLON));
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCheck.java:157]: Good job, @michaelmdelcavalcanti! You proactively applied the feedback from the last pull here. :clap:

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCheck.java [hunk 3]
```diff
@@ -87,7 +101,7 @@ private Map<String, String> _getMap(String fileName) throws Exception {
 		String[] lines = StringUtil.splitLines(FileUtil.read(importsFile));
 
 		for (String line : lines) {
-			int separatorIndex = line.indexOf("=");
+			int separatorIndex = line.indexOf(StringPool.EQUAL);
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCheck.java:104]: Good job, @michaelmdelcavalcanti! In the next pulls, please try to separate this kind of SF in a separated commit :wink:

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaCheck.testjava [hunk 4]
```diff
@@ -45,9 +45,32 @@ import com.liferay.document.library.kernel.model.DLFileEntry;
...
+import com.liferay.frontend.data.set.view.FDSView;
+import com.liferay.frontend.data.set.view.table.BaseTableFDSView;
+import com.liferay.frontend.data.set.view.table.FDSTableSchema;
+import com.liferay.frontend.data.set.view.table.FDSTableSchemaBuilder;
+import com.liferay.frontend.data.set.view.table.FDSTableSchemaBuilderFactory;
+import com.liferay.frontend.data.set.view.table.FDSTableSchemaField;
 
 /**
- * @author Nícolas Moura
+ * @author Michael Cavalcanti
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaCheck.testjava:56]: @michaelmdelcavalcanti, let's keep both authors :wink:

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaCheck.testjava [hunk 5]
```diff
@@ -45,9 +45,32 @@ import com.liferay.document.library.kernel.model.DLFileEntry;
...
+	public FDSTableSchema getFDSTableSchema() {
+		FDSTableSchemaBuilder fDSTableSchemaBuilder = _fDSTableSchemaBuilderFactory.create();
+
+		FDSTableSchemaField nameField = fDSTableSchemaBuilder.addFDSTableSchemaField("name", "name");
+
+		return fDSTableSchemaBuilder.build();
+	}
+
+	@Reference
+    private FDSTableSchemaBuilderFactory _fDSTableSchemaBuilderFactory;
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaCheck.testjava:75]: @michaelmdelcavalcanti, can you please format this code according to Liferay patterns? I mean, add tabulation, add blank line, etc

Brian may complain about it even it being a test dependency file

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaCheck.testjava [hunk 6]
```diff
@@ -45,9 +45,32 @@ import com.liferay.portlet.documentlibrary.model.DLFileEntry;
...
+import com.liferay.frontend.taglib.clay.data.set.ClayDataSetDisplayView;
+import com.liferay.frontend.taglib.clay.data.set.view.table.BaseTableClayDataSetDisplayView;
+import com.liferay.frontend.taglib.clay.data.set.view.table.ClayTableSchema;
+import com.liferay.frontend.taglib.clay.data.set.view.table.ClayTableSchemaBuilder;
+import com.liferay.frontend.taglib.clay.data.set.view.table.ClayTableSchemaBuilderFactory;
+import com.liferay.frontend.taglib.clay.data.set.view.table.ClayTableSchemaField;
 
 /**
- * @author Nícolas Moura
+ * @author Michael Cavalcanti
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaCheck.testjava:56]: @michaelmdelcavalcanti, let's keep both authors :wink:

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaCheck.testjava [hunk 7]
```diff
@@ -45,9 +45,32 @@ import com.liferay.portlet.documentlibrary.model.DLFileEntry;
...
+	public ClayTableSchema getClayTableSchema() {
+		ClayTableSchemaBuilder clayTableSchemaBuilder = _clayTableSchemaBuilderFactory.create();
+
+		ClayTableSchemaField nameField = clayTableSchemaBuilder.addClayTableSchemaField("name", "name");
+
+		return clayTableSchemaBuilder.build();
+	}
+
+	@Reference
+    private ClayTableSchemaBuilderFactory _clayTableSchemaBuilderFactory;
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaCheck.testjava:75]: @michaelmdelcavalcanti, can you please format this code according to Liferay patterns? I mean, add tabulation, add blank line, etc

Brian may complain about it even it being a test dependency file

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by michaelmdelcavalcanti** [comment]
ci:test:sf

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaCheck.testjava [hunk 8]
```diff
@@ -45,9 +45,36 @@ import com.liferay.document.library.kernel.model.DLFileEntry;
...
+	@Override
+	public FDSTableSchema getFDSTableSchema() {
+		FDSTableSchemaBuilder fDSTableSchemaBuilder = _fDSTableSchemaBuilderFactory.create();
+		FDSTableSchemaField nameField = fDSTableSchemaBuilder.addFDSTableSchemaField("name", "name");
+
+		return fDSTableSchemaBuilder.build();
+	}
+
+	@Reference
+    private FDSTableSchemaBuilderFactory _fDSTableSchemaBuilderFactory;
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaCheck.testjava:78]: @michaelmdelcavalcanti, can you please add a tab here?

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaCheck.testjava [hunk 9]
```diff
@@ -45,9 +45,36 @@ import com.liferay.portlet.documentlibrary.model.DLFileEntry;
...
+	@Override
+	public ClayTableSchema getClayTableSchema() {
+		ClayTableSchemaBuilder clayTableSchemaBuilder = _clayTableSchemaBuilderFactory.create();
+		ClayTableSchemaField nameField = clayTableSchemaBuilder.addClayTableSchemaField("name", "name");
+
+		return clayTableSchemaBuilder.build();
+	}
+
+	@Reference
+    private ClayTableSchemaBuilderFactory _clayTableSchemaBuilderFactory;
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaCheck.testjava:78]: @michaelmdelcavalcanti, can you please add a tab here?

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Pull request closed without merging**